### PR TITLE
OneTime Scans added!

### PR DESCRIPTION
**Pull request by Vesnyx**
Tootooroo~~

Didn't add icon, but I think everything else works.
Also forgot to update the meta userscript file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +29,4 @@ Screenshot: ![](http://i.imgur.com/UAjBySK.png)
 * Death Toll Scans
 * Easy Going Scans
 * Whiteout Scans
+* OneTime Scans
```
Comment (DakuTree) [inline README.md]: This should be "One Time Scans!", as that is the same as what they show on their site.

File: application/models/Tracker/Sites/OneTimeScans.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?php declare(strict_types=1); defined('BASEPATH') OR exit('No direct script access allowed');
+
+class OneTimeScans extends Base_Site_Model {
+	public $titleFormat   = '/^[a-z0-9_-]+$/';
+	public $chapterFormat = '/^en\/[0-9]+(?:\/[0-9]+(?:\/[0-9]+(?:\/[0-9]+)?)?)?$/';
+
+	public function getFullTitleURL(string $title_url) : string {
+		return "https://onetimescans.com/foolslide/series/{$title_url}";
```
Comment (DakuTree) [inline application/models/Tracker/Sites/OneTimeScans.php]: `onetimescans.com` doesn't exist, it should be `otscans.com`

File: application/views/Help.php [hunk 3]
```diff
@@ -47,5 +47,6 @@
 		<li>Death Toll Scans</li>
 		<li>Easy Going Scans</li>
 		<li>Whiteout Scans</li>
+		<li>OneTime Scans</li>
```
Comment (DakuTree) [inline application/views/Help.php]: Same as above, this should be "One Time Scans!".

File: public/CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,10 @@ Although this tries to adhere to "Semantic Versioning", this is a website, not a
 ## [Unreleased]
 - N/A
 
+## [1.7.10] 2017-07-05
+### Added
+- Support for OneTimeScans
```
Comment (DakuTree) [inline public/CHANGELOG.md]: Same as above, this should be "One Time Scans!".

**Comment by Vesnyx** [comment]
Woops, I made a ton of mistakes when I wrote this. Can't believe I didn't catch these when testing.

**Pull request merged**